prow/config.yaml

```diff
@@ -248,11 +248,37 @@ branch-protection:
         contexts:
         - cla/linuxfoundation
       repos:
+        apiserver-builder:
+          required_status_checks:
+            contexts:
+            - continuous-integration/travis-ci
         bootkube:
+          required_pull_request_reviews:
+            required_approving_review_count: 1
```

Comment:
Also here.

Reply:
ok, another vote for https://github.com/kubernetes/test-infra/issues/9451

Reply:
We need to resolve this before merge, right?

Reply:
this is resolved now that #9684 is in and I've rebased